dev/WindowsAppRuntime_BootstrapDLL/MddBootstrapActivity.h

```diff
@@ -73,17 +79,45 @@ namespace WindowsAppRuntime::MddBootstrap::Activity
             return m_lastFailure;
         }
 
-        void SetMddBootStrapAPI(MddBootStrapAPI mddBootStrapAPI)
+        void SetMddBootstrapAPI(MddBootstrapAPI mddBootstrapAPI)
+        {
+            m_mddBootstrapAPI = mddBootstrapAPI;
+        }
+
+        void SetInitializeData()
         {
-            m_mddBootStrapAPI = mddBootStrapAPI;
+            ++g_initializationCount;
         }
 
-        void SetDDLMFindMethodUsed(DDLMFindMethod ddlmFindMethodUsed)
+        // Returns a boolean to indicate to the caller if MddBootstrap should be shutdown
+        bool ShouldShutdownNow()
```

Comment:
Technically we're doing shutdown, it's just a question of the work involved

SUGGESTION: Rename to `DecrementInitializationCount()` and add a comment about returning true if the caller needs to do the cleanup work for shutdown to un-initialize our state